annotationProcessor/src/main/java/org/corfudb/annotations/ObjectAnnotationProcessor.java

```diff
@@ -562,6 +593,47 @@ private void addUpcallMap(TypeSpec.Builder typeSpecBuilder, TypeName originalNam
                 .build());
     }
 
+    private void addConflictExtractorMap(TypeSpec.Builder typeSpecBuilder, TypeName originalName,
+                                         Set<SMRMethodInfo> methodSet) {
+        // Generate the upcall string and associated map.
+        String conflictExtractorString = methodSet.stream()
+                .filter(x -> x.hasConflictAnnotations)
+                .map(x -> "\n.put(\"" + getSMRFunctionName(x.method) + CONFLICT_METHODS_SUFFIX + "\", " +
```

Comment:
Let me see if I understood you correctly. You're saying instead of generating ConflictFn for all signatures that have a conflict parameter,  generate and use a single generic method than can extract the conflict parameters given (method signature, Object[]) for signatures that have conflictParameterFunction.equals(""), is this correct?